data/notices.yaml

```diff
@@ -0,0 +1,19 @@
+# This file is used to create temporary notices on pages on an as needed basis.
+# For example when something is going to be deprecated, but has not been yet.
+# To use it, add a YAML object matching the following structure:
+#
+# - title: "This must match the page title"
+#   note: |
+#     A line, or multi-line message with details.
+#
+#     The note can be plain text, or use Markdown.
+#   type: One of NOTICE, WARNING, or DEPRECATED
+#
+# The 'type' field will render the div with the selfsame class applied.
+
+- title: "Image Overview: jre"
+  note: |
+    This image will switch from including a full compliment of JDK binaries to a minimally required set in a forthcoming release. The work is being tracked in [this GitHub issue](https://github.com/chainguard-images/images/issues/297).
```

Comment:
Should it be "complement" instead of compliment?

Reply:
Thanks for catching that!